code/modules/uplink/uplink_items.dm

```diff
@@ -1807,8 +1785,6 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	restricted_roles = list("Clown")
 	illegal_tech = FALSE
 
-//SKYRAT EDIT REMOVAL BEGIN
-/*
 /datum/uplink_item/role_restricted/spider_injector
```

Comment:
We removed those after they caused countless tickets every single time, because there is _always_ someone taking it too far.

Reply:
For this one Goof, it's more that a bunch of spiders would be "friendly" and obedient to their sentience-giver, then a bunch more would mindlessly RDM, then someone takes it out on the "friendly" ones, and before you know it, one spider has dragged a fuel tank into the bar in hopes of recreating recent scenes in Kabul and there are five tickets and now the rest of the round is trying to figure out who's just salting extremely hard and who's doing wrong.

It was just a headache and never really contributed that much to the round in the first place. It'd be better if it instead just spawned hostile spiders and be abundantly clear what their role is.